Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -368,6 +371,39 @@ def download_artifact(url, dir, pr)
     end
   end
 
+  def pr_check_conflicts(name, repo, pr)
+
+    pull_requests = search_issues("org:#{name}", repo: tap_remote_repo, state: "open", label: "\"long build\"")
+    long_prs = pull_requests.map { |pull_request| pull_request["number"] }
+    long_build_pr_files = []
+    long_prs.each do |long_pr|
+      GitHub.get_pr_files(name, tap_remote_repo, long_pr).each do |pr_file|
+        long_build_pr_files.append({ "file" => pr_file["filename"], "long_pr" => long_pr })
+      end
+    end
+
+    this_pr_files = GitHub.get_pr_files(name, tap_remote_repo, pr).map(&:filename)
+
+    conflicts = []
+    this_pr_files.each do |this_pr_file|
+      long_build_pr_files.each do |long_build_pr_file|
```

Comment:
`long_build_pr_files` should be a `Hash` rather than an `Array` with the filenames as keys so you can do a faster, key-based lookup rather than having to iterate through it.

Reply:
Done